src/faucet/Faucet.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import axios from 'axios';
+import ServiceFactory from '../services/ServiceFactory';
+import transactionMapper from '../addresses/TransactionMapper';
+import FaucetBlock from './LeftBlock';
+
+// import { broadcast, sponsorship, transfer, waitForTx } from '@acryl/acryl-transactions'
+
+// export const MASTER_SEED = 'test acc 2'
+// const API_BASE = 'https://nodestestnet.acrylplatform.com'
+
+// const ttx = transfer({ recipient: '3JHj9mVz1URexGe4MHCCEMBm4gJTkhqJnv5', amount: 270000000, feeAssetId: null }, MASTER_SEED)
+// broadcast(ttx, API_BASE)
+import TransactionList from '../addresses/TransactionList';
+
+export default class Faucet extends React.Component {
+    state = {
+        transactions: [],
+        address: '3JNbXos1asm3ZG47QG1MWjknQhvMDqrrH2v'  
+    }
+    setAcryl(str) {
```

Comment:
whats it?

Reply:
setAcryl use for replace WAVES name asset on ACRYL